CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.0] - 2020-08-24
```

Comment:
I don't quite understand - why 1.0.0?

Reply:
This was required by batman to bump the major version as this is a breaking change.

Reply:
@MarcelMue do you want me to increase the minor only or can I go ahead?